core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -490,6 +490,15 @@ public SELF withEnv(String key, String value) {
      * {@inheritDoc}
      */
     @Override
+    public SELF withEnv(Map<String, String> env) {
```

Comment:
you can do it as default method on `Container` interface as well since it's just a shortcut :)